platform/cog-platform-drm.c

```diff
@@ -248,6 +324,19 @@ init_drm (void)
         }
     }
 
+    if (g_getenv("WPE_DRM_CURSOR")) {
+        bool cursorSupported = drmSetClientCap(drm_data.fd, DRM_CLIENT_CAP_UNIVERSAL_PLANES, 1) == 0;
+        if (cursorSupported) {
+            if (initialize_cursor(drm_data.fd)) {
+                printf("cursor initialized\n");
+            } else {
+                printf("cursor failed to initialize\n");
+            }
+        } else {
+            printf("cursor not supported\n");
```

Comment:
Messages should be reported through `g_debug` or `g_warning`, depending on severity.